.github/workflows/ci.yml

```diff
@@ -152,6 +152,7 @@ jobs:
           # if [ -d $HOME/scp-firmware ]; then _make PLATFORM=stm32mp1-157C_DK2 CFG_SCMI_SCPFW=y CFG_SCP_FIRMWARE=$HOME/scp-firmware; fi
           _make PLATFORM=stm32mp2
           if [ -d $HOME/scp-firmware ]; then _make PLATFORM=stm32mp2 CFG_SCMI_SCPFW=y CFG_SCP_FIRMWARE=$HOME/scp-firmware; fi
+          if [ -d $HOME/scp-firmware ]; then _make PLATFORM=stm32mp2-235F_DK CFG_SCMI_SCPFW=y CFG_SCP_FIRMWARE=$HOME/scp-firmware; fi
```

Comment:
`CFG_SCMI_SCPFW=y` is necessary until https://github.com/OP-TEE/optee_os/pull/7529 is merged, 
Until the other PR is merged, to have a valid CI build for this PR I added the flag here, but it should be removed before merge to be clean

Reply:
Done